flit_core/flit_core/wheel.py

```diff
@@ -189,6 +189,14 @@ def copy_module(self):
             rel_path = osp.relpath(full_path, source_dir)
             self._add_file(full_path, rel_path)
 
+    def add_pth(self):
+        pth_target = osp.dirname(osp.abspath(self.module.path))
+        pth_file = self.module.path.name + ".pth"
+        with tempfile.NamedTemporaryFile("w") as f:
+            f.write(pth_target)
+            f.flush()
+            self._add_file(f.name, pth_file)
```

Comment:
You could use `with self._write_to_zip(pth_file):` here, I think - this is what we use elsewhere for generated files.